.golangci.yml

```diff
@@ -20,13 +20,13 @@ run:
   # default value is empty list, but next dirs are always skipped independently
   # from this option's value:
   #     vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
-  skip-dirs: [src,pkg,tmp,bin]
+  skip-dirs: [pkg,tmp,bin]
 
   # which files to skip: they will be analyzed, but issues from them
   # won't be reported. Default value is empty list, but there is
   # no need to include all autogenerated files, we confidently recognize
   # autogenerated files. If it's not please let us know.
-  skip-files: [student_test.go]
+  skip-files: [student_test.go,mysqlClient_test.go,mysqlClient.go,mongoClient.go,mongoClient_test.go]
```

Comment:
You can simply set this [this](https://github.com/abdulmoeid7112/student-backend-portal/blob/8db8904dd12dabbea52734c42c022310404961c9/.golangci.yml#L13) to false.

No need to specify tests files all the time.